packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -1,31 +1,88 @@
 import React from 'react';
-import { Tabs } from '@deriv/components';
+import { Tabs, Tab } from '@deriv/components';
 import { localize } from '@deriv/translations';
+import Chart from 'Components/chart';
+import DashboardComponents from './dashboard-components';
+import ReactJoyride from 'react-joyride';
+import { DBOT_ONBOARDING } from './joyride-config';
+import BotBuilder from './bot-builder';
+import classNames from 'classnames';
+import RunStrategy from './dashboard-components/run-strategy';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import Sidebar from './dashboard-components/sidebar';
+import RunPanel from '../run-panel';
+import QuickStrategy from './quick-strategy';
 
-const Dashboard: React.FC = () => {
-    const [active_index, setActiveTabIndex] = React.useState(0);
+interface DashboardProps {
+    active_tab: number;
+    setActiveTab: (active_tab: number) => void;
+    toggleStrategyModal: () => void;
+    is_drawer_open: boolean;
+}
+
+const Dashboard = ({ active_tab, setActiveTab, toggleStrategyModal, is_drawer_open }: DashboardProps) => {
+    const [show_side_bar, setShowSideBar] = React.useState<boolean>(true);
```

Comment:
```suggestion
    const [show_side_bar, setShowSideBar] = React.useState(true);
```
(same for the next line)
@prince-deriv primitive types can be inferred by useState based on the passed value, so no need to explicitly declare them here😊